src/graphql/queries/ListArticles.js

```diff
@@ -548,6 +560,25 @@ export default {
       });
     }
 
+    if (filter.articleContributesFrom) {
+      filterQueries.push({
+        nested: {
+          path: 'contributors',
+          query: {
+            bool: {
+              must: [
+                {
+                  term: {
+                    'contributors.userId': filter.articleContributesFrom.userId,
+                  },
+                },
+              ],
+            },
+          },
```

Comment:
We only have `contributors.userId` to match, so we can simplify this to
```suggestion
          query: {
            term: {
              'contributors.userId': filter.articleContributesFrom.userId,
            },
          },
```